deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml

```diff
@@ -89,6 +89,12 @@ objects:
     - spiaccesstokendataupdates
     verbs:
     - "*"
+  - apiGroups:
+    - appstudio.redhat.com
+    resources:
+    - remotesecrets
+    verbs:
+    - "*"
```

Comment:
I would love to stop using the `*` sign in RBAC. I understand that it's used for other rules, feel free to keep it here. But please keep in mind that we should ideally avoid using it in the future and also fix it in the ADR.

Reply:
Agreed, but as you correctly guessed, I was just going with the flow here..